packages/core/src/threatComposer/messageHandlers/addFileWatchMessageHandler.ts

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Command, FileChangedMessage, MessageType, WebviewContext } from '../types'
+import vscode from 'vscode'
+import { fsCommon } from '../../srcShared/fs'
+
+/**
+ * Function to add a watcher on the file that was opened. The watcher will notify Threat Composer
+ * view when a change occurs to the file externally.
```

Comment:
This is confusing. "Watchers" typically refer to filesystem watchers, but it looks like this is only using `onDidChangeTextDocument`.

It's *good* that filesystem watchers are avoided (they need to be used carefully, see also `packages/core/src/shared/fs/watchedFiles.ts` ). But the naming and description here is very confusing.